### PR TITLE
private/pedro/formcontrols treeview border fix

### PR DESCRIPTION
**Pull request by pedropintosilva**
- Fix drop-down marker position, form controls
- Improve drop-down marker cosmetics, form controls
- Fix drop-down marker (form control) size
- Do not add border to lists if dialog is a popmenu

### CODE REVIEW

**Comment by pedropintosilva** [comment]
dropdown marker good to review but the last commit is still a WIP

File: browser/src/control/Control.JSDialog.js [hunk 1]
```diff
@@ -260,7 +264,7 @@ L.Control.JSDialog = L.Control.extend({
 				windowId: data.id,
 				mobileWizard: this,
 				map: this.map,
-				cssClass: 'jsdialog' + (isAutofilter ? ' autofilter' : ''),
+				cssClass: 'jsdialog' + (isAutofilter ? ' autofilter' : '') + (isOnlyChild ? ' border-0 margin-0' : ''),
```
[the remark this answers is not in the document]
Comment (pedropintosilva) [reply browser/src/control/Control.JSDialog.js:267]: @eszkadev yeah, I was undecided what the best approach would be. I think my approach would be better if we were talking solely about border-0 but now it has also margin making it more convoluted and harder to follow in the future for no good reason. Will update. Thanks

[the remark this answers is not in the document]
Comment (eszkadev) [reply browser/src/control/Control.JSDialog.js:267]: (I mean: use custom class when there is more than one property set together at the same time as a whole)
combinations of params in one class -> ok
single param css rule -> useless I think

**Review by Rash419 — APPROVED** [review]
Thanks css part looks good : ) thanks

**Pull request merged**